src/sentry/api/base.py

```diff
@@ -160,6 +161,20 @@ def load_json_body(self, request):
         except json.JSONDecodeError:
             return
 
+    def _create_api_access_log(self):
```

Comment:
Maybe being a bit paranoid, but it might be worth just wrapping this in a try/catch to make sure this logging never breaks a request

Reply:
I actually really like this. I'll log an error in the api access log so we can see what went wrong.